pkg/config/validation/validation_test.go

```diff
@@ -121,6 +121,35 @@ func TestValidateWildcardDomain(t *testing.T) {
 	}
 }
 
+func TestValidateTrustDomain(t *testing.T) {
+	tests := []struct {
+		name string
+		in   string
+		out  string
```

Comment:
s/out/err
to be more accurate.

Reply:
Are you meaning: name -> s, in -> out, out -> err or errStr?

Reply:
i mean out-> err or errStr.